### ISSUE TITLE
Canvas import of CC 1.x package with webcontent resource marked as syllabus doesn't use it as syllabus

### ISSUE THREAD

**Issue by CreepyGnome** [instructure/canvas-lms#1142]
When you import a standard CC 1.x package into Cavnas that has a webcontent resource that has an intendeduse of syllabus the expectation is that resource will become the syllabus for the course. However it does not, the file is visible in the Files area of the course however that is not the expected results.
I have tried it with resource type of `webcontent` and `associatedcontent/imscc_xmlv1p3/learning-application-resource `which are supposed to be equivalent according to the CC spec.

Also according to the spec if you do not support a feature of the CC you must provide a warning. There is no warning stating you are not using the resource marked for use as a syllabus. 

So do you support the CC spec for webcontent with an intendeduse flag? The valid values are:

‘`assignment`’, ‘`lessonplan`’, ‘`syllabus`’, or ‘`unspecified`’

Using type webcontent:
```
    <resource identifier="59f0b40651800d0b3434408d_syllabus" type="webcontent" href="webcontent/syllabus.html" intendeduse="syllabus">
      <file href="webcontent/syllabus.html" />
    </resource>

```
Using type associatedcontent/imscc_xmlv1p3/learning-application-resource:
```
    <resource identifier="59f0b40651800d0b3434408d_syllabus" type="associatedcontent/imscc_xmlv1p3/learning-application-resource" href="webcontent/syllabus.html" intendeduse="syllabus">
      <file href="webcontent/syllabus.html" />
    </resource>
```

**Comment by CreepyGnome** [comment]
Here is an official IMS Global sample cartridge that has a Syllabus as a resource of type `webcontent `and the `intendeduse `set to `syllabus`.  It imported without warnings and without the file being used as the syllabus for the course.


[cc1p3vtd14v1p0.imscc.zip](https://github.com/instructure/canvas-lms/files/1415770/cc1p3vtd14v1p0.imscc.zip)

**Comment by omarkhan** [comment]
Thanks for the report @CreepyGnome. We are aware of this issue. No timeframe for a fix yet, but it's in our backlog.

**Comment by omarkhan** [comment]
Fixed in 4513d99
